### ISSUE TITLE
Unable to open update errors file.

### ISSUE THREAD

**Issue by dadanda** [tachiyomiorg/tachiyomi#6956]
### Steps to reproduce

1. Update the library.
2. Wait for it to finish.
3. Click the notification for "X update(S) failed)".
4. Open with "HTML Viewer".
5. Page results in "Webpage not available. The webpage at content://eu.kanade.provider/ext_cache_files/tachiyomi_update_errors.txt could not be loaded because : net::ERR_FILE_NOT_FOUND"

Using any other method to open the file such as Browsers or document viewer, send similar issues, such as cannot convert or file missing.

### Expected behavior

After step 4, it should open up the text file with the errors.

### Actual behavior

It results in file being missing.

### Tachiyomi version

0.13.3

### Android version

Android 11

### Device

Black Shark 4 (XIaomi) (PRS-H0)

### Other details

Specific Android version: 11 PROS2203060OS00MP5

I wasn't able to check the errors ever since it was installed on my phone.
I have not done this: https://tachiyomi.org/help/faq/#is-there-any-way-to-improve-performance
I have tried to use Shizuku to no avail

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Seems to be a common issue on MIUI devices. Closing in favor of #4616
